aiohue/v2/controllers/groups.py

```diff
@@ -86,12 +86,9 @@ def get_lights(self, id: str) -> List[Light]:
         return []
 
     async def set_state(self, id: str, on: bool = True) -> None:
-        """
-        Set supported feature(s) to grouped_light resource.
-
-        NOTE: a grouped_light can only handle OnFeature
-        To send other features, you'll have to control the underlying lights
-        """
+        """Set supported feature(s) to grouped_light resource."""
+        # Sending (color) commands to grouped_light was added in Bridge version 1.50.1950111030
+        self._bridge.config.require_version("1.50.1950111030")
```

Comment:
Should we make this a decorator ?

Reply:
Yeah I was thinking in that direction too but we require the config object so we'll have to fuss with a lookup from the class instance... Maybe for later ?